### ISSUE TITLE
[Feature Request] CLI manual scanner

### ISSUE THREAD

**Issue by Sanaki** [libretro/RetroArch#10736]
For frequently updated sets, being able to directly trigger the manual scanner via CLI would be particularly useful. This is especially true of things that can't be auto-scanned, such as the FBNeo console sets, hack and/or homebrew sets, and unlisted consoles.

**Comment by i30817** [comment]
Makes sense, i'd like to trigger the manual playlist scanner programmatically to (re)create the WHDLoad playlist i make with my (autoupdated) Whdload.dat.
